CHANGELOG.md

```diff
@@ -6,11 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [v1.0.1]
+### Changed
+- GDAL 3.x Support [#96](https://github.com/geotrellis/gdal-warp-bindings/pull/96)
+
+## [v1.0.1] - 2020-04-22
 ### Fixed
 - Fix Empty Metadata retrieval [#93](https://github.com/geotrellis/gdal-warp-bindings/pull/93)
 
-## [v1.0.0]
+## [v1.0.0] - 2020-03-12
 ### Added
 - Option to disable log message colorization [#78](https://github.com/geotrellis/gdal-warp-bindings/issues/78)
 - Add a special error code for the max attempts acceded case [#83](https://github.com/geotrellis/gdal-warp-bindings/issues/83)
```

Comment:
Small typo: It looks like "acceded" should be "exceeded"

Reply:
@jamesmcclain oops, typos :/